### ISSUE TITLE
Creating lot of data

### ISSUE THREAD

**Issue by ferpega** [Microsoft/mssql-docker#19]
Hello I am trying to create a huge amount of data (with a nodejs aplication launching scripts against the SQL Server dockerized).

But almost every time, after a while, I got this error.  Do you have any idea about what the problem can be?  Do you need something more?

```
sqlservr: ObjectWaitContext.cpp:533: void ObjectWaitContext::Wait(const PLONG64): Assertion `m_state == State::Signaled' failed.
Capturing core dump and information for sqlservr (pid 124)...
cat: core.sqlservr.124.temp/log/debug.log: No such file or directory
dmesg: read kernel buffer failed: Operation not permitted
No journal files were found.
No journal files were found.
ls: cannot access '/var/opt/mssql/log/log/errorlog*': No such file or directory
ls: cannot access '/var/opt/mssql/log/log/exception.log': No such file or directory
ls: cannot access '/var/opt/mssql/log/log/SQLDu*.txt': No such file or directory
ls: cannot access '/var/opt/mssql/log/log/SQLDu*.mdmp': No such file or directory
ls: cannot access '/var/opt/mssql/log/log/system_health*': No such file or directory
Packaging core dump and information into compressed files.
Core dump and information will be written to /var/opt/mssql/log/core.sqlservr.01_18_2017_09_13_11.124 and logs in /var/opt/mssql/log/core.sqlservr.01_18_2017_09_13_11.124.log
```

**Comment by ferpega** [comment]
The docker container is crashing now without load any data.  Only start it and wait a while then it crash with the same error noted above.

**Comment by twright-msft** [comment]
@ferpega This is a known bug that is fixed in the upcoming CTP 1.2 release.  Should be released on Friday of this week.  Sorry for the inconvenience.

**Comment by twright-msft** [comment]
We have released CTP 1.2 today that should fix this.  Please docker rmi microsoft/mssql-server-linux and then docker pull microsoft/mssql-server-linux to get the latest and verify that the issue is resolved.
